### PR TITLE
Added FieldRangeService (#69)

### LINKED ISSUE #69: Implement FieldRange Service
- create
- delete
- update

### PR DESCRIPTION
**Pull request by zzeropilot**

### CODE REVIEW

**Review by AndriiKret — DISMISSED** [review]

**Review by dtymkiv — DISMISSED** [review]

File: src/app/services/field_range.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    def get_by_range_id(range_id=None):
+        """
+        FieldRange model get by range id method
+
+        :param range_id: id of the range object
+        :return: list of FieldRange objects
+        """
+        if range_id is not None:
+            result = FieldRange.query.filter(FieldRange.range_id == range_id).all()
+        return result
```
Comment (IvanKostiv) [inline src/app/services/field_range.py]: Why we need default None?

File: src/app/services/field_range.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        """
+        FieldRange model update method
+
+        :param field_id: id of the field object
+        :param range_id: id of the range object
+        :return: updated FieldRange instance or None
+        """
+        field_range = FieldRangeService.get_by_field_id(field_id)
+        if not field_range:
+            raise FieldRangeNotExist()
```
Comment (yurdosii) [inline src/app/services/field_range.py]: Same checking for field_range equals to None in update and delete methods differ

File: src/app/services/field_range.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    def delete(field_id):
+        """
+        FieldRange model delete method
+
+        :param field_id: id of the field object
+        :return: True if instance was deleted or None
+        """
+        field_range = FieldRangeService.get_by_field_id(field_id)
+        if field_range is None:
+            raise FieldRangeNotExist()
```
Comment (yurdosii) [inline src/app/services/field_range.py:81]: Same checking for field_range equals to None in update and delete methods differ

File: src/app/services/field_range.py (hunk 1 above)
Comment (zzeropilot) [reply src/app/services/field_range.py]: Fixed

File: src/app/services/field_range.py (hunk 2 above)
Comment (zzeropilot) [reply src/app/services/field_range.py]: Fixed

**Review by IvanKostiv — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Review by yurdosii — APPROVED** [review]

**Pull request merged**